client/components/banner/test/index.jsx

```diff
@@ -1,165 +1,181 @@
-jest.mock( 'calypso/blocks/dismissible-card', () => {
-	return function DismissibleCard() {
-		return null;
-	};
-} );
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import preferencesReducer from 'calypso/state/preferences/reducer';
+import { renderWithProvider } from 'calypso/test-helpers/testing-library';
+import { Banner } from '../index';
 
-jest.mock( 'calypso/lib/analytics/track-component-view', () => {
-	return function TrackComponentView() {
-		return null;
-	};
-} );
+jest.mock( 'calypso/my-sites/plan-price', () => ( props ) => (
```

Comment:
Uber minor nit: We could destructure `props` here to directly use only the props we need. Like this:

```suggestion
jest.mock( 'calypso/my-sites/plan-price', () => ( { rawPrice } ) => (
```